views/topic/topic.hbs

```diff
@@ -144,12 +144,7 @@
                                 <small data-toggle="tooltip" title="{{$t "topic._topic.label.expandTasks"}}"><i
                                         class="fa fa-chevron-right collapse-icon" aria-hidden="true"></i></small>
                                 {{$t "global.headline.assignedTasks" }}
-                                <small>({{arrayLength homeworks}}
-                                    {{#ifneq 1 (arrayLength homeworks)}}
-                                        {{$t "global.headline.tasks"}}
-                                    {{else}}
-                                        {{$t "global.headline.task"}}
-                                    {{/ifneq}})
+                                <small>({{arrayLength homeworks}})
```

Comment:
Why was the headline deleted here?

Reply:
Assigned tasks (1 task) vs  Assigned tasks (1)
From a UX point of view, it is sufficient to only specify the number. Please also consider a mobile view.